packages/zarith/zarith.1.7/opam

```diff
@@ -33,7 +33,9 @@ depends: [
   "ocamlfind"
   "conf-gmp"
   "conf-perl" {build}
+  "conf-gcc" {build}
 ]
+patches: [ "absolute_CC.patch" ]
```

Comment:
I think we need to bump the version of zarith here rather than adding this patch unconditionally. Perhaps `zarith.1.7a`.  Otherwise if it causes issues for other users, they cannot revert to the old version easily.

Reply:
`zarith.1.7-1` would be more standard I think